py2pack/templates/fedora.spec

```diff
@@ -3,13 +3,13 @@
 Name:           python-%{pypi_name}
 Version:        {{ version }}
 Release:        %autorelease
-Summary:        {{ summary }}
+Summary:        {{ summary|replace('\n','') }}
```

Comment:
I think it's not needed to modify the templates, you have defined the new `fix_data` function, what if we do this sanitize operation there instead of doing it in the templates? We can add to the `fix_data` function something like:

```python
    single_line_fields = ['summary', 'license', 'home_page', 'source_url']
    for field in single_line_fields:
        if field not in data_info:
            continue
        # remove line breaks to avoid multiline rpm spec file
        data_info[field] = data_info[field].replace('\n', ' ')
```